src/content/en/updates/2017/09/sensors-for-the-web.md

```diff
@@ -349,17 +349,24 @@ through HTTPS. During development you can do so via http://localhost but for pro
 you'll need to have HTTPS on your server. See Security with [HTTPS article](/web/fundamentals/security/)
 for best practices and guidelines there.
 
-### Only the main frame
+### Feature Policy integration
 
-To prevent iframes from reading sensor data the Sensor objects can be created
-only within a main frame.
+The [Feature Policy integration](https://w3c.github.io/sensors/#feature-policy-api) in Generic Sensor API is used to control access to sensors data for a frame.
```

Comment:
please wrap lines at 80 chars (and throughout)

Reply:
done